Client/Cards/actions/actionTest.ts

```diff
@@ -0,0 +1,55 @@
+import * as pax2pay from "../../../index"
+import { ErrorResponse } from "../../../model"
+import * as model from "../../../model"
+import { Card } from "../../Card"
+
+export async function actionTest(card: Card & model.CardResponse, client: pax2pay.Client) {
+	if (!card.providerCardId || !card.providerCode)
+		throw Error("Card id or provider code missing.")
+	else if (!card.fundingAccount)
+		throw Error("Funding account missing.")
+	else {
+		const amendedCard = await card.amend({
+			newBalance: 2,
+			currency: card.fundingAccount.currency,
+		})
+		const amendedActualCard = await client?.cards.getCard(card.providerCardId, card.providerCode)
+
+		if (ErrorResponse.is(amendedActualCard) || ErrorResponse.is(amendedCard))
+			throw Error("Error after amending card")
```

Comment:
you could do like ErrorResponse.assert() and have it just fail the tests in that function, rather than manually doing it each time